src/com/projectkorra/projectkorra/airbending/combo/AirSweep.java

```diff
@@ -42,6 +42,7 @@
 	private Vector direction;
 	private ArrayList<Entity> affectedEntities;
 	private ArrayList<BukkitRunnable> tasks;
+	private double hitRadius;
```

Comment:
We typically just have this variable called `radius` in our other moves. Might be worth changing it to that just for consistency's sake.

Reply:
Alright, I’ll change it to radius.